### ISSUE TITLE
Installation of yices on Mac os fails

### ISSUE THREAD

**Issue by smover** [pysmt/pysmt#412]
The installation of yices on Mac os fails, since pymst dowloads the archive for linux (yices-2.5.1-x86_64-unknown-linux-gnu-static-gmp.tar.gz)

```
$> pysmt-install --yices
This script allows you to install the solvers supported by pySMT.

By executing this script, you confirm that you have read and agreed
with the licenses of each solver.

Notice: the installation process might require building tools
        (e.g., make and gcc).

Continue? [Y]es/[N]o: Y
Downloading: /Users/smover/.smt_solvers/yices/yices-2.5.1-x86_64-unknown-linux-gnu-static-gmp.tar.gz Bytes: 5791477
100.0%    
./install-yices: line 99: break: only meaningful in a `for', `while', or `until' loop
./install-yices: line 135: break: only meaningful in a `for', `while', or `until' loop

Installation failed

Can't find /sbin/ldconfig


running build_ext
building '_yicespy' extension
swigging yices_python.i to yices_python_wrap.c
swig -python -I/Users/smover/.smt_solvers/python-bindings-2.7/yices_bin/include -o yices_python_wrap.c yices_python.i
yices_python.i:216: Error: Unable to find 'yices_types.h'
yices_python.i:217: Error: Unable to find 'yices.h'
error: command 'swig' failed with exit status 1
Traceback (most recent call last):
  File "/usr/local/bin/pysmt-install", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/pysmt/cmd/install.py", line 240, in main
    installer.install(force_redo=options.force_redo)
  File "/usr/local/lib/python2.7/site-packages/pysmt/cmd/installers/base.py", line 143, in install
    self.compile()
  File "/usr/local/lib/python2.7/site-packages/pysmt/cmd/installers/yices.py", line 68, in compile
    self.install_yicespy()
  File "/usr/local/lib/python2.7/site-packages/pysmt/cmd/installers/yices.py", line 58, in install_yicespy
    wheel_file = glob.glob(os.path.join(self.base_dir, "yicespy") + "*.whl")[0]
IndexError: list index out of range
```

**Comment by mikand** [comment]
@smover Thanks for the report. Indeed we never really tested pysmt-install under OSX.

By any chance do you have some time to work on this issue? (Of course, we can provide support and guidance)

**Comment by marcogario** [comment]
You can take a look at https://github.com/pysmt/pysmt/blob/master/pysmt/cmd/installers/msat.py to see how to change the installer to get the osx version. 

Before you start changing that, have you tried manually installing yices in mac? Is there any additional step or library that might be needed? If the issue is only the download, then it should be reasonably easy to change, and we would appreciate if you could try it out. 

Please note that we are not running the last version of yices. If you want you can open a separate pull request with the updated version number in install.py.  This will run our tests and let us know if it works on linux.

**Comment by marcogario** [comment]
Closed as duplicate of #96 . 
#96 contains the status of the installers for OSX.
#328 contains the status of the installers for Windows.

We might want to make this info more prominent in the README, or extend pysmt-installer to warn if the OS is currently not supported.
